yb-voyager/cmd/common.go

```diff
@@ -394,13 +400,23 @@ func displayImportedRowCountSnapshot(state *ImportDataState, tasks []*ImportFile
 			utils.ErrExit("failed to get imported snapshot rows map: %v", err)
 		}
 	}
+	keys := make([]sqlname.NameTuple, 0, len(snapshotRowCount.Keys()))
```

Comment:
isn't just this enough to get the keys
```
keys := snapshotRowCount.Keys()
```

Reply:
snapshotRowCount.Keys returns string[]. I cannot use the string object to look up in the snapshotRowCount in the slice function.